Update apps.md

coderofsalvation opened the pull request:
added aframe-remotestorage

raucao commented:
Hi there, and thanks for the addition. A couple of comments:

1. The app is using a RS category/directory named "myapp". You should choose a category name that describes what the app is storing in it, for example `webxr`, `webxr-scenes`, or just `aframe` to make it app-specific.
2. As a user with an RS account, it's not obvious to me how I can load or save anything after launching this app and connecting my storage.

coderofsalvation commented:
> 1. The app is using a RS category/directory named `myapp`. You should choose a category name that describes what the app is storing in it, for example `webxr`, `webxr-scenes`, or just `aframe` to make it app-specific.

I would not worry too much about this.
It's just a boilerplate/demo (hence it's added to the tutorial/demo section in the PR).
Imho a 'myapp' placeholder is fine.
Apps based on this boilerplate would not benefit from having different app-data ending up in the same folder.

**UPDATE**: changed my mind. Since it's literally a one-line commit, I decided to rename `myapp` to `aframe`. It might even be useful to have such a directory with snippets, which can be used across different apps.

> 2. As a user with an RS account, it's not obvious to me how I can load or save anything after launching this app and connecting my storage.

Thanks, I've added an [explanation in the repo](https://github.com/c-frame/aframe-remotestorage?tab=readme-ov-file#demo-explanation) and a link to the repo in the demo.

raucao commented on apps.md:
```diff
@@ -86,6 +86,7 @@ may be outdated.
 | [My Favorite Drinks](https://myfavoritedrinks.remotestorage.io) | Keep a list of your favorite drinks | myfavoritedrinks | [GitHub](https://github.com/RemoteStorage/myfavoritedrinks) | Simple demo app, maintained by rs.js devs |
 | [Hello](https://hello.0data.app) | Simple Hello World with remoteStorage, Solid, and Fission | todos  | [GitHub](https://github.com/0dataapp/hello) | Works well on mobile and desktop |
 | [Lucchetto Onboard](https://overhide.github.io/armadietto/lucchetto/onboard.html) | Enables in-app purchase SKU onboarding for [luchetto.js](https://www.npmjs.com/package/lucchetto/v/latest) extended RS apps | pay2my.app | [GitHub](https://github.com/overhide/armadietto/blob/master/lucchetto/onboard.html) | wide screens only; featured in the [remote-storage tutorial](https://github.com/overhide ...
+| [AFRAME XR component](https://aframe-remotestorage.5apps.com) | Startingpoint to load/store (Web)XR scenes from/to remoteStorage | VR/AR | [Codeberg](https://codeberg.org/coderofsalvation/aframe-remotestorage) | works on mobile, desktop & VR/AR headsets |
```
I guess "launch the AFRAME designer (ctrl+shift+i)" doesn't work for you either?

On my machine (and by default with any other Linux I suppose), this shortcut opens the normal dev console in browsers.

DougReeder replied:
Launching the A-Frame Inspector now works for me.  The instructions are kinda buried, though — consider how they could be more prominent.

Separately, the data should probably be saved as type 'text/html' rather than `text/plain'.

DougReeder reviewed (approved):
More documentation in the app would help, but the app is good enough to list.

raucao commented:
I'm still unable to open the inspector, and thus create any changes to be saved. Is it really supposed to be the same shortcut as the developer console?

coderofsalvation commented:
![image](https://github.com/user-attachments/assets/e0fd73eb-7ebe-4086-9485-334afb87228c)

> I'm still unable to open the inspector, and thus create any changes to be saved. Is it really supposed to be the same shortcut as the normal browser devtools?

afaik ctrl+shift+i (AFRAME inspector) is not the same as ctrl+shift+j (browser console).
Anyways, I've added an editor-button (and mentioned it in README.md too).

I'm out of resources to spent more on this, so I hope we've been able to strike the balance between audit & 'sample code'.

raucao commented:
Yes, ctrl-shift-i is the same as "inspect page", i.e. opening browser dev tools (at least in Firefox and Chrome on Linux, but I assume also on Windows).

I can open the editor via the button now. :+1:
